### ISSUE TITLE
Real Solar System

### ISSUE THREAD

**Issue by AC1016** [jrossignol/ContractConfigurator#714]
Exception occured while loading contract 'unknown':
System.ArgumentException: 'Kerbin' is not a valid CelestialBody.
  at ContractConfigurator.ConfigNodeUtil.ParseCelestialBodyValue (System.String celestialName) [0x00046] in <ef0243a06f2841fe9bf57034a334902e>:0 
  at ContractConfigurator.ConfigNodeUtil.ParseSingleValue[T] (System.String key, System.String stringValue, System.Boolean allowExpression) [0x00179] in <ef0243a06f2841fe9bf57034a334902e>:0 
  at ContractConfigurator.ConfigNodeUtil.ParseValue[T] (ConfigNode configNode, System.String key, System.Boolean allowExpression) [0x002bc] in <ef0243a06f2841fe9bf57034a334902e>:0 
  at ContractConfigurator.ConfigNodeUtil.ParseValue[T] (ConfigNode configNode, System.String key, T defaultValue) [0x00012] in <ef0243a06f2841fe9bf57034a334902e>:0 
  at ContractConfigurator.ConfiguredContract.OnLoad (ConfigNode node) [0x00121] in <ef0243a06f2841fe9bf57034a334902e>:0

**Comment by jrossignol** [comment]
Sounds like you have some bad data in your save. Does this happen on a new
save as well?

**Comment by AC1016** [comment]
> 

yep it happens on a new save
i cant even quit to main menu
this happens when i download rss

**Comment by jrossignol** [comment]
Yeah, but what contract packs?  Not sure why the error is not giving a
contract name, but it's a specific contact that it's giving you this issue
(and a LOT of them are not explicitly RSS compatible).
